test/get-newlines-between-option.test.ts

```diff
@@ -0,0 +1,205 @@
+import { describe, expect, it } from 'vitest'
```

Comment:
Can you move this file to `test/utils/`?

Reply:
@azat-io Oops, rebase from https://github.com/azat-io/eslint-plugin-perfectionist/pull/425 didn't move it 😅. Moved in f4b4ad5060d0c87da7621928c4d73e5ac8c53644.

Reply:
Looks good to me!